src/pyseekdb/cli.py

```
@@ -0,0 +1,311 @@
+                table.add_column("Count", justify="right", style="magenta")
+
+                for coll in collections:
+                    try:
+                        count = str(coll.count())
+                    except Exception:
+                        count = "N/A"
+                    table.add_row(coll.name, str(coll.dimension or "N/A"), count)
+
+                console.print(table)
+                console.print(f"\n[dim]Total: {len(collections)} collection(s)[/dim]")
+    except Exception as e:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)